nvflare/app_common/aggregators/intime_accumulate_model_aggregator.py

```diff
@@ -24,7 +24,8 @@
 
 class InTimeAccumulateWeightedAggregator(Aggregator):
     def __init__(self, exclude_vars=None, aggregation_weights=None, expected_data_kind=DataKind.WEIGHT_DIFF):
-        """Perform accumulated weighted aggregation
+        """Perform accumulated weighted aggregation.
+
```

Comment:
not sure if these changes going to conflict with #165

Reply:
This class should behave the same after #165 is merged.  That PR does not change the API of this class.  However, there are some sentences been rephrased.  I can resolve the conflicts if this one is merged after #165.